packages/ui/components/SlideOver.tsx

```diff
@@ -0,0 +1,35 @@
+interface SlideOverProps {
+  isOpen: boolean;
+  onClose: () => void;
+  children: React.ReactNode;
+}
+
+const CloseIcon: React.FC = () => (
+  <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M6 18L18 6M6 6L18 18" stroke="#94A3B8" strokeWidth="1.5" strokeLinecap="round" strokeLinejoin="round"/>
+  </svg>
+);
```

Comment:
```suggestion
```

You already have a close icon here: https://atoms.asyncapi.com/?path=/docs/design-tokens-icons--docs#icons. Just import it from `ui/icons`.